tox.ini

```diff
@@ -1,36 +1,33 @@
 [tox]
 envlist =
-    {py35,py36,py37,py38,py39,pypy3}-sphinx{201,212,222,231,244,300,321,latest}
-    {py27, pypy2}-sphinx{185}
+    {py39,py310,py311,py312,py313,pypy3}-sphinx{621,747,latest},
+    wheel
 
 [testenv]
 minversion = 3.15.0
-whitelist_externals = make
+allowlist_externals = make
 commands =
-    ./setup.py bdist_wheel
-    ./setup.py install
     make -C docs html
 deps =
-    pip
-    setuptools
     sphinx-rtd-theme==0.1.6
-    sphinx185: sphinx==1.8.5
-    sphinx201: sphinx==2.0.1
-    sphinx212: sphinx==2.1.2
-    sphinx222: sphinx==2.2.2
-    sphinx231: sphinx==2.3.1
-    sphinx244: sphinx==2.4.4
-    sphinx300: sphinx==3.0.0
-    sphinx321: sphinx==3.2.1
     sphinxlatest: sphinx
-    wheel
+    # last release for old versions
+    sphinx747: sphinx==7.4.7
+    sphinx621: sphinx==6.2.1
+
+[testenv:wheel]
+minversion = 3.15.0
+commands =
+    python -m build --wheel .
```

Comment:
Since building the wheel doesn't need to be tested in all the python versions I split this out, and only test that it builds with the latest Python (at least on the CI), if you run locally I believe this will use the base Python (what you have installed).